registration/templates/registration/welcome_email.txt

```diff
@@ -1,26 +1,24 @@
 {% autoescape off %}
-Hi {{ application.user.profile.full_name }},
+Hi!
 
-We are pleased to inform you that your application to participate in the
-OpenCraft Instance Manager beta test has been accepted.  Your Open edX instance
-has been provisioned and is available at
+Thank you very much for applying for a free 30-day trial of our self-hosting professional & teacher plan. You seem to have an interesting project - this is the type of initiative we are looking to help with this offer. 
 
-    {{ application.instance.url }}
+Your instance is provisioned and ready to use:
 
-The Studio interface used to create course materials is available at
+{{ application.instance.url }}
 
-    {{ application.instance.studio_url }}
+The Studio interface used to create course materials is available at:
 
-To log in, use the email address {{ application.user.email }} and the password
-you provided in the application.  You can reset your password by clicking the
-"Forgot password?" link on the login page
+{{ application.instance.studio_url }}
 
-    {{ application.instance.url }}/login
+To log in, use the email address {{ application.user.email }} and the password you provided in the application. If you don't remember it, you can reset your password by clicking the "Forgot password?" link on the login page:
 
-Since the OpenCraft Instance Manager is in beta state, there might still be some
-problems or downtime for your instance.  We appreciate any feedback on the
-project – just reply to this email!
+{{ application.instance.url }}/login
+
+The professional & teacher self-hosting plan offers you a stable and powerful dedicated Open edX instance. If you would like to customize your site, or have access to great features like larger data storage, guaranteed availability, paid courses, mobile apps or access to the Insights course analytics engine, please ask me about our institutional offering. Your questions are most welcome.
```

Comment:
+1, except for "99.9% guaranteed availability" - we don't provide a specific availability, except for clients who request it.

Reply:
@antoviaque I mention that since it's better to be more specific than guaranteed availability which sounds like 100%. The [full version of the document](https://drive.google.com/file/d/1DvK6gUFTfep6HbWDmBteNXJ0wJkp2YWd/view) does have that number though.

Reply:
Since the paragraph instructs the client to get in touch with us to ask about the guaranteed availability, I think that can be left out to be discussed directly.